api/commands/run.ts

```diff
@@ -1,9 +1,11 @@
 import { Credentials } from "@dataform/api/commands/credentials";
 import * as dbadapters from "@dataform/api/dbadapters";
 import { retry } from "@dataform/api/utils/retry";
+import { hashExecutionAction } from "@dataform/api/utils/run_cache";
 import { dataform } from "@dataform/protos";
 import * as EventEmitter from "events";
 import * as Long from "long";
+import * as _ from "lodash";
```

Comment:
please name this import

Reply:
thats the official name :D anyway, changing it to Lodash